===================================================================
RCS file: /home/cvs/linux/include/asm-mips/unistd.h,v
retrieving revision 1.59
retrieving revision 1.60
diff -u -p -r1.59 -r1.60
--- linux/include/asm-mips/unistd.h	2004/04/08 15:40:01	1.59
+++ linux/include/asm-mips/unistd.h	2004/04/16 22:51:13	1.60
@@ -1049,6 +1049,7 @@ type name (atype a,btype b,ctype c,dtype
 
 #include <linux/compiler.h>
 #include <linux/types.h>
+#include <linux/linkage.h>
 #include <asm/ptrace.h>
 #include <asm/sim.h>

